### PR TITLE
RyuTas - A Ryujinx TAS workspace.

### PR DESCRIPTION
**Pull request by Shrek5InTheatres2019**
I thought having a built in TAS Workspace/utility would be nice. Right now, this is just in the idea/planning phase, with some writing code intermittent, but I thought i'd open this for discussion.

### CODE REVIEW

**Comment by Thog** [comment]
That could be a great idea! Have you talked with the people at TASBot about it? It would be nice to brainstorm something that adapt to everyone need 👍

**Comment by GalladeGuy** [comment]
It'd be a good idea to look at these guidelines on TASVideos if you haven't already:
[Requirements](http://tasvideos.org/EmulatorResources/Requirements.html)
[Features](http://tasvideos.org/EmulatorResources/Features.html)

**Comment by Shrek5InTheatres2019** [comment]
Wrote up some things on how this thing might work https://docs.google.com/document/d/12jX2LC1C2LA7o10dYBwg4L8RyHrQqVozuDF9I8LpBsE/edit?usp=sharing , feel free to comment changes or whatever, i didn't open it up to editing as internet people are idiots.

File: Ryujinx/Ui/MainWindow.cs [hunk 1]
```diff
@@ -113,6 +115,8 @@ private MainWindow(Builder builder) : base(builder.GetObject("_mainWin").Handle)
                 End(null);
             }
 
+            builderr = builder;
```
Comment (Shrek5InTheatres2019) [inline Ryujinx/Ui/MainWindow.cs:118]: is this an alright way to get access to DOM style things? Is there a better way for me to make the TAS topbar menu visible/not in code?

**Comment by Shrek5InTheatres2019** [comment]
https://shrekk.is-inside.me/byUJei9u.png that happens when you load a game rn, not sure why. will figure out, maybe.

**Comment by HorrorTroll** [comment]
Nah, that just an issue from OpenTK. I alway have that issue when load a games, try restart Ryujinx and run again

**Comment by Shrek5InTheatres2019** [comment]
Yeah, I restarted a few times. It seems to happen regardless. :(

**Pull request closed without merging**